docs/ReleaseNotes.md

```diff
@@ -1,6 +1,22 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 2.3.2
+Acuminator 2.3.2 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Enhancements
+In Acuminator 2.3.2, the following enhancements have been implemented:
+ - The [PX1076](diagnostics/PX1076.md) diagnostic has been added to check for and warn a developer that the PXInternalUseOnly attribute is used in the customization code.
+ - The documentation for the [PX1046](diagnostics/PX1046.md) diagnostic was extended, the reasoning for the diagnostic is now provided.
+ 
+### Fixed Bugs
+In this version of Acuminator, the following bugs have been fixed:
+ - Due to the changes in the theme updating mechanism, an error was thrown by Visual Studio in the Acuminator colorizer, and the Acuminator coloring became corrupted.
+ - Due to the changes in the latest Visual Studio updates, Acuminator custom settings were not enabled, and the Acuminator analyzers used default values.
```

Comment:
I think we should specify Visual Studio version here. Also the root cause of the issue should be specified for this bug. I would say it like this:
`Acuminator custom code analysis settings specified by the user did not work with a new 64-bit out of process static code analysis introduced by Visual Studio 2019. When out of process feature was enabled only default settings were used.`